### ISSUE TITLE
Transition pending for weeks and now funds gone

### ISSUE THREAD

**Issue by ejpcmac** [iotaledger/wallet#357]
Back to version 2.3.1, I sent about 143 MIOTA in my wallet. It was all good.

Then arrived 2.4.0 and its transition process. Please let me state this sentence from the transition window:

> For the transition phase your entire balance will be moved to a newly generated address under the same seed.

I was asked my seed, then the balance was OK and I started the migration. When asked for a seed, I provided the same, as stated in the previous sentence. The transaction was then shown in my wallet, but pending for more than a month, with a balance showing 0 —but the transaction was somewhere.

Now, I upgraded to 2.5.1, and there is no more transaction shown. I tried to attach to the tangle, but it says the hash is invalid. Same here on [iotasearch](https://iotasear.ch/hash/EIEKNFWBFJZTFMZPVKEOYCRSNLJRIFDA9SBKQHDERCGKAOQREIYQTHAQCVAL9NUWRP9A9BXFAZXRTVGUYPPGKTUDED).

So, what am I supposed to do to get my IOTAs back?

**Comment by lowiet01** [comment]
I had the same issue where my IOTA did not reflect in my IOTA Light Wallet (v. 2.5.1), upon being successfully transferred from Bitfinex. When I had set the wallet up, I guess I wasn't giving much consideration to the node choice. I used the following path: Tools > Edit Configuration. I then noticed that one of the Host options (from the drop-down menu) was associated with Bitfinex (I had another non-Bitfinex option selected, but I don't recall which one it was)--I chose the http://iota.bitfinex.com:80 option. I left the Min Weight Magnitude at '15' and the Curl Implementation as 'Webgl 2 Curl implementation.' Upon choosing changing this setting, my IOTA instantly appeared. Hopefully, this will work for some who've encountered the same problem.

**Comment by lvella** [comment]
Similar issue here. I used the same seed, and I don't remember if the transitioning was confirmed or not, but after updating to 2.5.1, my balance was lost. Using Python API, I've generated 500 addresses from index 0, and 60 addresses for each index from 0 to 99, all empty.

**Comment by OnosToolan** [comment]
Yeah i had 2 transactions pending. On another thread for my problem some said to install wallet again. So did that and now my entire wallet is 0

So much anger with this wallet. Where is the help file to recover all lost tokens please?

**Comment by OnosToolan** [comment]
How can IOTA just confiscate tokens? Where is the authority to do that?

**Comment by wonkytonky** [comment]
snapshots are agreed with by community
